File: php/easypay.lib/Easypay_config.php
<?php

namespace easpaylib;

class Easypay_config
{
    private $_basepath;

    public final function get_basepath()
    {
        return $this->_basepath;
    }

    public final function set_basepath($value)
    {
        $this->_basepath = $value;
    }

    private $_easypay_secret;

    public final function get_easypay_secret()
    {
        return $this->_easypay_secret;
    }

    public final function set_easypay_secret($value)
    {
        $this->_easypay_secret = $value;
    }

    private $_host;

    public final function get_host()
    {
        return $this->_host;
    }

    public final function set_host($value)
    {
        $this->_host = $value;
    }

    private $_merchant_id;

    public final function get_merchant_id()
    {
        return $this->_merchant_id;
    }

    public final function set_merchant_id($value)
    {
        $this->_merchant_id = $value;
    }
}

File: Android/EasypayClient/clientlibrary/src/main/java/com/swisscom/easypay/clientlibrary/EasypayConfig.java
package com.swisscom.easypay.clientlibrary;

public class EasypayConfig {
    private String Basepath;
    public final String getBasepath()
    {
        return Basepath;
    }
    public final void setBasepath(String value)
    {
        Basepath = value;
    }
    private String EasypaySecret;
    public final String getEasypaySecret()
    {
        return EasypaySecret;
    }
    public final void setEasypaySecret(String value)
    {
        EasypaySecret = value;
    }
    private String Host;
    public final String getHost()
    {
        return Host;
    }
    public final void setHost(String value)
    {
        Host = value;
    }
    private String MerchantId;
    public final String getMerchantId()
    {
        return MerchantId;
    }
    public final void setMerchantId(String value)
    {
        MerchantId = value;
    }

}
